test/verify/check-metrics

```diff
@@ -1094,6 +1094,31 @@ BEGIN {{
         m.execute("dd if=/dev/zero of=/var/cockpittest/blob bs=1M count=40")
         b.wait(lambda: progressValue(self, ".pf-c-progress[data-disk-usage-target='/var/cockpittest']") >= 90)
 
+        # read/write in popover
+        b.click("#current-metrics-card-disks .all-disks-no-gap button")
+        b.wait_visible(".pf-c-popover .disks-nowrap")
+        # read
+        m.execute("systemd-run --collect --slice cockpittest --unit disk-read-hog sh -ec 'while true; do echo 3 > /proc/sys/vm/drop_caches; grep -r . /usr >/dev/null; done'")
```

Comment:
This is very taxing. Could you please move the checks here into the already existing "reading lots of data" section above? It would fit very well there, and avoid testing the same scenario twice.